commitlint.config.js

```diff
@@ -15,9 +15,10 @@ module.exports = {
         'scope-min-length': [2, 'always', 3],
         // Body
         'body-min-length': [2, 'always', 30],
-        'body-max-line-length': [2, 'always', 100],
+        'body-max-line-length': [2, 'always', 130],
```

Comment:
Same here, why is it changed?

Reply:
I want it bigger, checked with my 13" screen, to see when github gonna move to the next line.